app/assets/stylesheets/frontend/views/_html-publications.scss

```diff
@@ -286,6 +286,71 @@
           }
         }
       }
+
+      table.financial-data {
+
+        .numeric {
+          text-align: right;
+          font-family: $NTA-Light-Tabular;
+        }
+
+        // make all elements inside thead look the same
+        // (td can sometimes be used to overcome complexity of multiple lines in thead)
+        // and make all of thead and tfoot stand out
+        thead,
+        tfoot {
+          background-color: #e1e8e8;
```

Comment:
Can we just use `$grey-3` here? It's close enough:

![screen shot 2016-10-28 at 11 40 34](https://cloud.githubusercontent.com/assets/1650875/19803940/596e93d0-9d03-11e6-9ed8-ac15d0b467b8.png)